cni/pkg/ebpf/common.h

```diff
@@ -0,0 +1,51 @@
+#ifndef __COMMON_H__
+#define __COMMON_H__
+
+#define ENABLE_IPV4
+
+
+
+#define OUTBOUND_CB (4321)
+#define INBOUND_CB {1234}
+#define ZTUNNEL_BYPASS_MARK (8888)
+
+
+#define ZTUNNEL_INBOUND_MARK (5678)
+#define ZTUNNEL_OUTBOUND_MARK (8765)
+
+#define ZTUNNEL_HBONE_PORT (15008)
+#define ZTUNNEL_INBOUND_PORT (15008)
+#define ZTUNNEL_INBOUND_PLAINTEXT_PORT (15006)
+#define ZTUNNEL_OUTBOUND_PORT (15001)
+
+#define MAX_APPS (65535)
+#define MAX_SERVER_PORTS_PER_APP (32)
+#define APP_INFO_MAP_SIZE MAX_APPS
+
+
+/* copy of 'struct ethhdr' without __packed */
+struct eth_hdr {
+    unsigned char   h_dest[ETH_ALEN];
+    unsigned char   h_source[ETH_ALEN];
+    unsigned short  h_proto;
+};
+
+struct ztunnel_info {
+    uint32_t        ifindex;
+#ifdef ENABLE_IPV4
+    uint32_t        ipv4;
```

Comment:
IP info for ztunnel is not necessarily now. Will remove it later

Reply:
removed